### ISSUE TITLE
Tenant aware scheduled commands and notifications

### ISSUE THREAD

**Issue by juhasev** [hyn/multi-tenant#707]
What is the recommended current approach to get tenancy working with scheduled tasks? The problem is that when the scheduled tasks runs it will use the system database connection as opposed to running scheduled tasks in all the tenant databases. It looks like the only way to do this via Artisan command, am I correct?

````
php artisan tenancy:run
````

Also it would nice to have out of box support for TenantAwareNotification. Right now the easiest way to get these working is to Queue notification as Tenancy supports Laravel Queues out of the box. Something like this:

````
<?php

namespace App\Notifications;

use Illuminate\Bus\Queueable;
use Illuminate\Notifications\Notification;
use Illuminate\Contracts\Queue\ShouldQueue;
use Hyn\Tenancy\Queue\TenantAwareJob;

class Announcement extends Notification implements ShouldQueue
{
    use Queueable;
    use TenantAwareJob;

    // Your code here
}

````

---

__Proposed behavior__

It would be nice to have something like this for quick Notifications like database notifications.
````
<?php

namespace App\Notifications;

use Illuminate\Notifications\Notification;
use Hyn\Tenancy\Queue\TenantAwareNotification;

class Announcement extends Notification 
{
    use TenantAwareNotification;

    // Your code here
}
````

---
__Information__

- hyn/multi-tenant version: 5.3
- laravel version: 5.7

**Comment by ArlonAntonius** [comment]
Should be implemented when #711 is working and merged.

**Comment by luceos** [comment]
Merged, fixed.
